cfgov/unprocessed/apps/rural-or-underserved-tool/js/show-map.js

```diff
@@ -39,7 +38,7 @@ function scriptLoaded( evt ) {
  * Set access map options and create map.
  */
 function initializeMap() {
-  window.L.mapbox.accessToken = mapboxAccessToken;
+  window.L.mapbox.accessToken = window.cfpb_mapboxAccessToken;
```

Comment:
The snake casing will likely be flagged by our JS linter. I'd just make it camel 🐫

Reply:
It doesn't flag it because it's set in jinja (which we don't run linting on) but I'll update it anyway.